dracut/anaconda-diskroot

```diff
@@ -24,7 +24,7 @@ run_checkisomd5() {
                 rc=$?
                 state="inactive"
             fi
-            if [ "$rc" == "1" ]; then
```

Comment:
This doesn't feel correct. However, the question here is what we would like to follow POSIX or bash? I think we should decide and use that consistently everywhere to avoid confusions. (Not saying in this PR).

Reply:
Yup, I commented on that earlier.